Makefile

```diff
@@ -460,8 +460,5 @@ generate_ci_workflows:
 generate-flag-testdata:
 	./tools/generate_flag_testdata.sh
 
-release-notes:
-	go run ./go/tools/release-notes --from "$(FROM)" --to "$(TO)" --version "$(VERSION)" --summary "$(SUMMARY)"
-
 install_kubectl_kind:
```

Comment:
is this unused?

Reply:
It was used previously, but now it isn't. Check the changes in the next file. It's asking the users to run the binary directly with `go run` instead of having a `make` command